pkgs/applications/science/biology/ants/default.nix

```
@@ -41,7 +42,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     for file in $out/bin/*; do
-      wrapProgram $file --set PATH "$out/bin"
+      wrapProgram $file --set PATH "$out/bin":${coreutils}/bin
```

I think this is the more technically correct way to go about this
```suggestion
      wrapProgram $file --prefix PATH : "$out/bin"
```

---

Changed to use `--prefix`.